src/routes/DesignSystemApp.svelte

```diff
@@ -349,9 +349,7 @@
 
   <DesignSystemPanel
     title="Radios"
-    code={`<script>
-      let radioValue = "none";
-   </script>
+    code={`<script ✂prettier:content✂="CiAgICAgIGxldCByYWRpb1ZhbHVlID0gIm5vbmUiOwogICA=">{}</script>
```

Comment:
This is a [bug in Prettier's Svelte Plugin](https://github.com/sveltejs/prettier-plugin-svelte/issues/70). We've removed that `<script/>` tag from there as a temporary workaround.